addons/l10n_hr/models/account_move.py

```diff
@@ -0,0 +1,128 @@
+import re
+
+from odoo import _, api, models, fields
+from odoo.exceptions import ValidationError
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_hr_process_type = fields.Selection(
+        [
+            ('P1', "P1: Issuing invoices for deliveries of goods and services according to purchase orders, based on contracts"),
+            ('P2', "P2: Periodic invoicing for deliveries of goods and services based on contracts"),
+            ('P3', "P3: Issuing invoices for delivery according to an independent purchase order"),
+            ('P4', "P4: Prepayment (advance payment)"),
+            ('P5', "P5: Payment on the spot (Sport payment)"),
+            ('P6', "P6: Payment before delivery, based on purchase order"),
+            ('P7', "P7: Issuing invoices with references to the delivery note"),
+            ('P8', "P8: Issuing invoices with references to the shipping and receipt notes"),
+            ('P9', "P9: Credits or invoices with negative amounts, issued for various reasons, including empty returns packaging"),
+            ('P10', "P10: Issuing a corrective invoice (reversal/correction of invoice)"),
+            ('P11', "P11: Issuing partial and final invoices"),
+            ('P12', "P12: Self-issuance of invoice"),
+            ('P99', "P99: Customer-defined process"),
+        ],
+        string="Business Process Type",
+        compute='_compute_l10n_hr_process_type',
+        store=True,
+        readonly=False,
+    )
+    l10n_hr_customer_defined_process_name = fields.Char(
+        string="Custom Process Name",
+        help="Required when Process Type is P99. Specify the name of your custom business process. "
+            "This will appear in the UBL as P99:YourProcessName",
```

Comment:
I'm not sure P99 as a default is a good idea. I wonder what good defaults would be? P1 for invoices and P10 for credit notes?
In any case, I think we can just change this to P1 for now.